src/userbase-server/user.js

```diff
@@ -2758,6 +2762,10 @@ const _createStripeUpdatePaymentMethodSession = async function (user, admin, cus
       },
       success_url,
       cancel_url,
+      automatic_tax: {
+        enabled: enableAutomaticTax,
+      },
+      allow_promotion_codes: allowPromotionCodes,
```

Comment:
Neither are needed here, Stripe fails with:

[1]: "You can not pass `allow_promotion_codes` in `setup` mode."
[2]: "You cannot set `automatic_tax[enabled]=true` in `setup` mode."

Tax and promo logic relies on what was set at time of first purchase, which I think is ok? Might be tricky to make the logic more complex